### ISSUE TITLE
Nautilus fails to open, traps error:0 in libglib2.0.so.0.6200.4

### ISSUE THREAD

**Issue by gorangrbic** [clearlinux/distribution#1686]
From dmesg:
[  927.448162] traps: nautilus[5932] trap int3 ip:7f619d8d2e8b sp:7ffcff0fd650 error:0 in libglib-2.0.so.0.6200.4[7f619d870000+b9000]
[  928.473527] traps: nautilus[6333] trap int3 ip:7fd59ec13e8b sp:7fff533c45b0 error:0 in libglib-2.0.so.0.6200.4[7fd59ebb1000+b9000]
[  931.577759] traps: nautilus[6479] trap int3 ip:7f1b430bde8b sp:7ffdcf378cc0 error:0 in libglib-2.0.so.0.6200.4[7f1b4305b000+b9000]
[  932.898164] traps: nautilus[6584] trap int3 ip:7fd697ac9e8b sp:7ffea79e6ee0 error:0 in libglib-2.0.so.0.6200.4[7fd697a67000+b9000]

Fresh install. Have only the following packages installed via Software Center:
cryptography
storage-utils
linux-tools
net-tools
performance-tools
strace
wireshark
flac
virt-manager
virt-manager-gui
kernel-native-dkms
Mark Text
Vorta
Teams for Linux
Bitwarden
ffmpeg
Atom

No apps have been removed. External SSD mounted at /home (and /etc/fstab file created) with regular User directories and no dot files/dirs aside from .ssh and a modified .bashrc to include a .bash_aliases.

I've been using the OS for a greater part of January and this issue only began appearing yesterday, after I attempted to organize the GNOME Applications menu. Today, I installed and subsequently uninstalled Fondo. Afterwards, Nautilus began throwing out the above error. 

Please let me know if you need additional info.

**Comment by gorangrbic** [comment]
The issue was resolved by reinstalling the desktop, desktop-apps, desktop-autostart packages.

I didn't save the original strace, but the last line in the strace output for Nautilus was "Settings schema org.freedesktop.Tracker.Miner.Files' is not installed". After a little bit of research, I found this thread: 
https://github.com/hhfeuer/Gentoo-Gnome-3.28-X/issues/9

I remembered removing the GNOME Screenshot Tool late last night and thought the issues might be related, so I reinstalled the desktop-apps with
```bash
sudo swupd bundle-remove desktop-apps --force
```
and
```bash
sudo swupd bundle-add desktop-apps desktop desktop-autostart
```
This resolved the issue. 

It appears that the primary cause is tracker being split into multiple packages (2018), which seems to still be the case. 
https://github.com/hhfeuer/Gentoo-Gnome-3.28-X/issues/9#issuecomment-412825961
